build/localnet/templates/v1-validator-template.yaml.tpl

```diff
@@ -29,7 +29,7 @@ spec:
             - -config=/configs/config.json
             - -genesis=/genesis.json
           ports:
-            - containerPort: 8080
+            - containerPort: 42069
               name: consensus
             - containerPort: 50832
               name: rpc
```

Comment:
Below this line there is a list of environment variables. Here is what I'm suggesting:
```
    - name: POCKET_P2P_HOSTNAME
       value: "v1-validator${VALIDATOR_NUMBER}"
```

That will override the configuration that we have in config.json. It works, because:
1. We follow The Twelve-Factor App methodology for [configuration files](https://12factor.net/config), and all our configuration parameters can be changed via environment variables which take priority over config.json.
2. We create a Service object below in the same file with this name.

Reply:
Thanks for explaining that makes much more sense!